documentation/modules/metrics/proc-strimzi-metrics-reporter-kafka-deploy-options.adoc

```diff
@@ -0,0 +1,64 @@
+// This assembly is included in the following assemblies:
+//
+// metrics/assembly_metrics-kafka.adoc
+
+[id='proc-metrics-reporter-kafka-deploy-options-{context}']
+= Enabling Strimzi Metrics Reporter metrics
```

Comment:
do we need metrics on the end or can we match with the JMX Exporter title: "Enabling Strimzi Metrics Reporter "